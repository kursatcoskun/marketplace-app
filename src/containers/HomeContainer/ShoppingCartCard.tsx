import { Card, CardProps } from "antd";
import React from "react";
import styled from "styled-components";

export const ShoppingCartCard: React.FunctionComponent<CardProps> = styled(
  Card
)`
  width: 296;
  height: auto;
  border: 8px solid #1ea4ce;
  background-color: #ffffff;
  border-radius: 2px;
  @media only screen and (max-width: 800px) {
    flex-direction: column;
  }
`;
