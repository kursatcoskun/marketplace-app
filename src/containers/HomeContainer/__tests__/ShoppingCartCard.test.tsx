// MyComponent.test.js
import { shallow } from "enzyme";
import React from "react";
import { ShoppingCartCard } from "../ShoppingCartCard";
describe("LeftBarCardComponent", () => {
  it("should render ShoppingCartCard component", () => {
    const wrapper = shallow(<ShoppingCartCard />);
  });

  it("should render initial layout", () => {
    const component = shallow(<ShoppingCartCard />);
    expect(component.getElements()).toMatchSnapshot();
  });
});
