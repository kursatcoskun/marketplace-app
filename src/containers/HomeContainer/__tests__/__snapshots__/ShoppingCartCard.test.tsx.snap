// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LeftBarCardComponent should render initial layout 1`] = `
Array [
  <Card
    className="sc-bdnxRM eNTeaF"
  />,
]
`;
